sdk/python/feast/feature_set.py

```diff
@@ -663,6 +660,41 @@ def is_valid(self):
         if len(self.entities) == 0:
             raise ValueError(f"No entities found in feature set {self.name}")
 
+    def update_schema(self, schema: Schema):
```

Comment:
I am a little bit concerned that `schema` is too generic. It might be confused for a feature set schema. Can we be more explicit here? `update_from_tfx_schema`?

Reply:
Updated the method names for working with Tensorflow metadata schema to `export_tfx_schema` and `import_tfx_schema`